src/helpers.ts

```diff
@@ -318,3 +318,26 @@ export function getLinkedEpics(timelineItems: ZenhubTimelineItem[]): ZenhubIssue
 
     return [...connectedEpics.values()];
 };
+
+/**
+ * Fetches a list of changed files and mark those that contain changes in test files.
+ */
+export async function isPullRequestTested(octokit: OctokitType, pullRequest: PullRequest) {
+    const files = await octokit.rest.pulls.listFiles({
+        owner: ORGANIZATION,
+        repo: pullRequest.base.repo.name,
+        pull_number: pullRequest.number,
+    });
+    const filePaths = files.data.map((file) => file.filename);
+    const testFilePaths = filePaths.filter((filePath) => {
+        return filePath.includes('/test/')
+            || filePath.includes('/tests/')
+            || filePath.endsWith('.test.js')
```

Comment:
It would serve regexp here as you can have a test in python or crawlee has in [test.mjs](https://github.com/apify/crawlee/pull/1944/files) 
`.*tests?\.\w{2,3}$`
https://regex101.com/r/6jdNGR/1

Reply:
Now I found that tests in python start with test_bla.py
Maybe improve in way
`(\.|\_|\w)*tests?(\.|\_|\w)*\.\w{2,3}$`
https://regex101.com/r/3AAUD8/1

Reply:
Cool, thanks. I also added a unit test